boards/arm/qemu_cortex_r5/board.cmake

```diff
@@ -4,6 +4,8 @@
 
 set(EMU_PLATFORM qemu)
 set(QEMU_ARCH xilinx-aarch64)
+board_set_flasher_ifnset(qemu)
+board_set_debugger_ifnset(qemu)
```

Comment:
Just add a boards/common/qemu.board.cmake which includes these lines, and include it from the board.cmake files, the same way we do for the other runners.